Maple2.Database/Model/Map/Nurturing.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Maple2.Database.Extensions;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Maple2.Database.Model;
+
+internal class Nurturing {
+    public long AccountId { get; set; }
+    public int ItemId { get; set; }
+    public long Exp { get; set; }
+    public short ClaimedGiftForStage { get; set; }
+    public DateTime CreationTime { get; set; }
+    public DateTime? LastFeedTime { get; set; }
```

Comment:
I don't think this needs to be null. We can just ensure the conversion is done properly.